zookeeper-contrib/zookeeper-contrib-zkpython/src/test/run_tests.sh

```diff
@@ -30,9 +30,9 @@ else
 fi
 
 # Find the build directory containing zookeeper.so
-SO_PATH=`find ../../../build/ -name "zookeeper.so" | head -1`
+SO_PATH=`find ../../build/ -name "zookeeper.so" | head -1`
```

Comment:
Isn't any 'target' directory name?
Did you build with raw 'make' the c client?

There are multiple ways to build the C client

Reply:
I used ant to build the c client as maven doesn't seem to be working (the c client is not being built even with "full-build" profile). That's the way it was always done on the past - just continuing until we have a maven based approach. So with this we get back to status quo - after the directory structure changes for maven I mean.

I did a "ant clean compile compile-native" at the toplevel, then "ant compile" and "ant test" within the zookeeper-contrib-zkpython directory.